### DISCUSSION TITLE
Missing rplidar_ros2 dependancy

### DISCUSSION THREAD

**Discussion by cdiehl6** [aws-deepracer/aws-deepracer-follow-the-leader-sample-project#7] [Q&A]
Hello, I have been running into an issue where it seems like the rplidar_ros2 package isn't available anymore. 

When I try to import it during step 7 of the download and build portion of the instructions, it asks me to login to github and then fails, saying "fatal: repository 'https://github.com/youngday/rplidar_ros2.git/' not found". When I go to the repository it doesn't exist, so I guess it probably is set to private or has been removed, unless I am doing something wrong. 

Do you gave a suggestion for getting the package or a replacement package? I haven't been able to find one. As it is the LIDAR package, it disrupts basically every project, I just happened to run into it on follow the leader.

**Comment by sirajulmuneer** [comment]
I am having the same issue as well. Any hint to resolve the issue?

[INFO] [launch]: Default logging verbosity is set to INFO
Task exception was never retrieved
future: <Task finished name='Task-2' coro=<LaunchService._process_one_event() done, defined at /opt/ros/foxy/lib/python3.8/site-packages/launch/launch_service.py:271> exception=PackageNotFoundError("package 'rplidar_ros2' not found, searching: ['/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/ftl_launcher', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/webserver_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/web_video_server', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/deepracer_systems_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/usb_monitor_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/status_led_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/ctrl_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/servo_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/sensor_fusion_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/rplidar_ros', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/object_detection_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/model_optimizer_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/inference_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/i2c_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/ftl_navigation_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/device_info_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/deepracer_navigation_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/camera_pkg', '/root/deepracer_ws/aws-deepracer-follow-the-leader-sample-project/deepracer_follow_the_leader_ws/install/deepracer_interfaces_pkg', '/opt/ros/foxy']")>
Traceback (most recent call last):
  File "/opt/ros/foxy/lib/python3.8/site-packages/ament_index_python/packages.py", line 49, in get_package_prefix
    content, package_prefix = get_resource('packages', package_name)
  File "/opt/ros/foxy/lib/python3.8/site-packages/ament_index_python/resources.py", line 48, in get_resource
    raise LookupError(
LookupError: Could not find the resource 'rplidar_ros2' of type 'packages'

During handling of the above exception, another exception occurred:
